src/css/objects/_media.scss

```diff
@@ -245,172 +241,216 @@
+      height: $width / 1.33333;
     }
 
+    .o-media__figure--widescreen {
+      height: $width / 1.77778;
+    }
   }
 
 }
 
-.media--hp-large {
+.o-media--lg {
   margin-top: $base-line-height;
 
-  > .media__content {
-    box-shadow: inset 17rem 0 0 $color-gray-lightest;
-    padding: $base-line-height 0 ($base-line-height * 2) 19rem;
+  > .o-media__content {
+    // box-shadow: inset 17rem 0 0 $color-gray-lightest;
+    // padding: $base-line-height 0 ($base-line-height * 2) 19rem;
 
     @include media-max($media-tablet) {
       box-shadow: none;
       padding-left: 0;
     }
-
   }
 
-  > .media__figure {
+  .o-media__figure {
     border-bottom: 6px solid $color-primary-alt;
-    margin: 0 ($gutter * -1);
+
+    // margin: 0 ($gutter * -1) //NOTE: Unclear why we need this.
     width: auto;
+    margin-bottom: 2.5rem;
 
     @include media-max($media-mobile) {
       margin: 0 ($gutter / -2);
     }
   }
 
-  > .media__headline {
+  > .o-media__headline {
     margin: 0 0 $base-line-height;
   }
 
-  .media__label {
+  .o-media__label {
     font-size: ms(-2);
   }
 
-  .media__overline {
+  .o-media__overline {
     color: $color-secondary;
   }
-
 }
 
-.media--horizontal {
-
-  .media__figure {
-    float: left;
-    margin-right: 3.5550459%;
-    width: 37.5%;
+.o-media--block {
+  .o-media__figure {
+    border-bottom: 4px solid $color-primary;
   }
+}
 
-  .media__bylineText {
-    border-bottom: 4px solid $color-gray-light;
-    padding-bottom: 0.5rem;
+.o-media--block-simple {
+  .o-media__headline {
+    margin-top: 0.5rem;
   }
+}
 
+.o-media--last {
+  margin-bottom: 0;
 }
 
-.media--horizontalSmall {
-  align-items: center;
-  display: flex;
-  margin: 0 0 $base-line-height;
+.o-media--new-and-unread {
+  .o-media__headline {
+    position: relative;
 
-  .media__figure {
-    margin-right: 1.125rem;
-    max-width: 5rem;
+    &::before {
+      background-color: $color-primary;
+      border-radius: 50%;
+      content: "";
+      display: block;
+      height: 0.5rem;
+      left: $base-line-height / -2;
+      position: absolute;
+      top: 1rem;
+      width: 0.5rem;
+    }
   }
 
-  .media__headline, .media__title {
-    margin: 0;
+  .o-media__indicator-visited {
+    display: none;
   }
 
-}
+  //NOTE: Not sure why I commented this out, but it doesn't appear to do anything now.
+  // &::after
+  //   border-top: 2px solid $color-gray-lightest
+  //   content: ''
+  //   display: block
 
-.media--block {
+  //   //NOTE: Was 17rem
+  //   left: 0
+  //   position: absolute
+  //   right: 0
+  //   top: 0
 
-  .media__figure {
-    border-bottom: 4px solid $color-primary;
+  //   +media-max($media-tablet)
+  //     display: none
+
+  > .o-media__figure {
+    margin: 0 0 $base-line-height;
   }
 
-}
+  > .o-media__headline {
+    margin: 0 0 $base-line-height;
+  }
 
-.media--blockSimple {
+  > .o-media__label {
+    font-size: ms(-2);
+  }
 
-  .media__headline {
-    margin-top: 0.5rem;
+  .o-media__overline {
```

Comment:
This override won't be necessary once you set the default overline color to $color-secondary.